src/Pdf/Engine/MpdfEngine.php

```diff
@@ -13,8 +14,15 @@ public function output()
     {
         //mPDF often produces a whole bunch of errors, although there is a pdf created when debug = 0
         //Configure::write('debug', 0);
+        $format      = $this->_Pdf->orientation() == 'landscape' ? $this->_Pdf->pageSize() . '-L' : $this->_Pdf->pageSize();
         $orientation = $this->_Pdf->orientation() == 'landscape' ? 'L' : 'P';
-        $MPDF = new \mPDF($this->_Pdf->encoding(), $this->_Pdf->pageSize() . '-' . $orientation);
+        $MPDF        = new \Mpdf\Mpdf([
+            'mode'        => $this->_Pdf->mode(),
```

Comment:
Given that the constructor take an array of options I don't see the need to add a new property/method for mode. Just use config options like Dompdf engine does https://github.com/FriendsOfCake/CakePdf/blob/0ec08ba62d4ca44928dadb8c925135a8855e3e4e/src/Pdf/Engine/DomPdfEngine.php#L21

Reply:
mode does not same encoding type. I think we need new one for prevent confusion.

Mode of the document. Is parsed from values of various formats:
     *  Codepage Values (case-insensitive)
     *  Country/Language code values (case-insensitive)